src/org-manager/store/effects/organisation.effects.ts

```diff
@@ -194,6 +194,15 @@ export class OrganisationEffects {
     })
   );
 
+
+  @Effect()
+  public navToReviewOrganisation$ = this.actions$.pipe(
+    ofType(pendingOrgActions.OrgActionTypes.NAV_TO_REVIEW_ORGANISATION),
+    map(() => {
+      return new fromRoot.Go({ path: ['/review-organisation'] });
+    })
```

Comment:
semicolon;

Reply:
same implementation with other effects, so to be consisted in remaining part of the code it left.